client.go

```diff
@@ -369,15 +379,18 @@ func (c *Client) openConn(idx int, host string) (pconn *persistentConn, err erro
 
 	var conn net.Conn
 
+	ctx := context.Background()
+	if c.config.Timeout != 0 {
```

Comment:
config.Timeout has a default so it will never be 0.

Reply:
Saw that. I’m not sure I agree with the design decision (versus defaulting timeout if zero at runtime), but it’s your library. Happy to change.

Reply:
Done (719b580). I also noted where the library should wire up a parent `Context` in the future to allow overall FTP operations to be governed by a caller’s `Context`.